python/pyspark/serializers.py

```diff
@@ -251,6 +256,36 @@ def __repr__(self):
         return "ArrowPandasSerializer"
 
 
+class ArrowStreamPandasSerializer(Serializer):
+    """
+    Serializes Pandas.Series as Arrow data with Arrow streaming format.
+    """
+
+    def load_stream(self, stream):
+        import pyarrow as pa
+        reader = pa.open_stream(stream)
+        for batch in reader:
+            table = pa.Table.from_batches([batch])
+            yield [c.to_pandas() for c in table.itercolumns()]
+
+    def dump_stream(self, iterator, stream):
```

Comment:
Maybe add few comments for `dump_stream` and `load_stream` like `ArrowPandasSerializer`.

Reply:
Sure, I'll add comments.